HCK06_2019_Janelia/projects/ExtensionSharing/README.md

```diff
@@ -18,28 +18,36 @@ Develop methods for sharing NWB:N extensions.
 
 ## Objectives
 
-- Evaluate plans for infratructure for web-based sharing, deployment, and evaluation of extensions
-- Begin implmetation of the online archive for NWB:N extensions
+- Evaluate plans for infrastructure for web-based sharing, deployment, and evaluation of extensions
+- Begin implementation of the online archive for NWB:N extensions
 
 ## Approach and Plan
 
 - Work with attendees to evaluate the current plan
 - Implement draft template repo for extensions
-- Implement draft template repo for extension recipies 
+- Implement draft template repo for extension recipes
 
 ## Progress and Next Steps
 
-- Created GitHub organization to develop the NWB:N extensions catalog 
+- Created GitHub organization to develop the NWB:N extensions catalog
 - Created draft GitHub webpage for the extension catalog
 - Created design documents for the extension catalog
-- Created draft extension and sharing and versioning guidelines
-- Added continuous integration (using Azure-pipelines) to [NeurodataWithoutBorders/nwb-docutils](https://github.com/NeurodataWithoutBorders/nwb-docutils), [nwb-extensions/ndx-template](https://github.com/nwb-extensions/ndx-template)
+- Created draft extension sharing and versioning guidelines
+- Added continuous integration (using Azure-pipelines) to [NeurodataWithoutBorders/nwb-docutils](https://github.com/NeurodataWithoutBorders/nwb-docutils), [nwb-extensions/ndx-template](https://github.com/nwb-extensions/ndx-template) and  [NeurodataWithoutBorders/pynwb](https://github.com/NeurodataWithoutBorders/pynwb)
 - Consolidated documentation of `ndx-template` with the one generated by `nwb-docutils`
 - Reviewed conda-forge infrastructure and identified next step for enabling continuous-integration of [nwb-extensions/staged-extensions](https://github.com/nwb-extensions/staged-extensions)
+- Discussed the flowchart for users creating and publishing their own extensions
+  - Decided to simplify the initial creation of an extension using a web form
+  - Decided that `version` needs to be required on YAML spec files, and now that MatNWB cache spec bug has been fixed, `cache_spec` should default to True when writing an NWB file.
```

Comment:
MatNWB cache spec bug has been fixed -> MatNWB cache spec issue has been fixed

Link to the issue report ?  or PR ?